src/crypto/CHIPCryptoPAL.h

```diff
@@ -36,9 +36,10 @@
 namespace chip {
 namespace Crypto {
 
-const size_t kP256_FE_Length     = 32;
-const size_t kP256_Point_Length  = (2 * kP256_FE_Length + 1);
-const size_t kSHA256_Hash_Length = 32;
+const size_t kP256_FE_Length                  = 32;
+const size_t kP256_ECDSA_Signature_Length_Raw = (2 * kP256_FE_Length);
+const size_t kP256_Point_Length               = (2 * kP256_FE_Length + 1);
```

Comment:
This is not a point length, it's the uncompressed public key representation (which is a point, but a particular version of it).

Suggest renaming these to match or be closer to the crypto primitives section of the spec, since this is CHIPCrypto module:

> 
> int CHIP_CRYPTO_GROUP_SIZE_BYTES := 32
> --
> int CHIP_CRYPTO_PUBLIC_KEY_SIZE_BYTES : = (2 * CHIP_CRYPTO_GROUP_SIZE_BYTES) + 1 = 65
> 

Feel free to remove "CHIP_" prefix, but reusing the names would be good.

Reply:
Added TODO to rename that in the future